{
  "Launchpad": "https://bugs.launchpad.net/bugs/1370999",
  "Id": "1370999",
  "Gerrit": "https://review.openstack.org/122100",
  "IdGerrit": "122100",
  "IdCommit": "d65ca49990f1241a5a8afdc3ad0e1a99c878275a",
  "Files": "nova/tests/virt/xenapi/test_agent.py,nova/virt/xenapi/agent.py,plugins/xenserver/xenapi/etc/xapi.d/plugins/agent,plugins/xenserver/xenapi/etc/xapi.d/plugins/xenstore.py",
  "Lines_Inseted": "51",
  "Lines_Deleted": "5",
  "CommitParent": "51de439a4d1fe5e17d59d3aac3fd2c49556e641b",
  "Classification": "Unknown",
  "KeywordsTitle": "unreliable, due to",
  "Description": "correctly updated, can cause, the need, we need, ",
  "KeywordsCommit": "changing,  might happen ,Until now , fetching,this change ensures,  wrong, Turns out,  has been updated.",
  "Comments": "I'm not sure if this is a bug report or not, because if the reboots is necesary and only they want take more care detecting reboots"
}